spec/features/subjects_taught_spec.rb

```diff
@@ -3,6 +3,8 @@
 RSpec.feature "Choosing subjects taught during Teacher Student Loan Repayments claims" do
   before do
     start_claim
+    choose_school schools(:penistone_grammar_school)
+
     visit claim_path("subjects-taught")
```

Comment:
Ah yeah, good call. I initially added the above line because we needed a claim_school to be able to show the subjects-taught page, but then we get redirected to that page anyway.